gobblin-data-management/src/main/java/gobblin/data/management/partition/CopyableDatasetRequestor.java

```diff
@@ -52,6 +53,8 @@
 @AllArgsConstructor
 @Getter
 public class CopyableDatasetRequestor implements PushDownRequestor<FileSet<CopyEntity>> {
+  private final static String CONF_PREFIX = CopyConfiguration.COPY_PREFIX + "." + CopyableDatasetRequestor.class.getSimpleName();
```

Comment:
Can you put the configuration key in `CopyConfiguration`? Also, a more descriptive and less specific key might be better. A suggestion might be `gobblin.copy.abortOnSingleDatasetFailure`.